experimental/torch_xla2/test/test_ops.py

```diff
@@ -269,6 +270,7 @@ def test_reference_eager(self, device, dtype, op):
         continue
       check_output = op.name not in random_ops
 
+      print("[DEBUG] sample_input: ", sample_input)
```

Comment:
thanks, do we want to keep these `print` code here?

Reply:
commenting this line out. But retaining the commented code for future debugging. This is a commonly used debug line.